lxd/api_1.0.go

```diff
@@ -417,5 +421,17 @@ func doApi10UpdateTriggers(d *Daemon, nodeChanged, clusterChanged map[string]str
 			return err
 		}
 	}
+
+	if candidChanged {
+		endpoint := clusterConfig.CandidEndpoint()
+		expiry := clusterConfig.CandidExpiry()
+		domains := clusterConfig.CandidDomains()
+
+		err := d.setupExternalAuthentication(endpoint, expiry, strings.Split(domains, ","))
```

Comment:
We should allow both "," and ", "

Reply:
Got this fixed locally, will push once the builders are done rebuilding their cache.